app/scripts/modules/core/src/presentation/Markdown.tsx

```diff
@@ -26,12 +26,12 @@ export class Markdown extends React.Component<IMarkdownProps> {
   public render() {
     const { message, tag, ...rest } = this.props;
 
-    if (!message) {
+    if (message == null) {
```

Comment:
does linter not yell at this?

Reply:
it does not. would you prefer the more explicit `=== null || === undefined`?

Reply:
no, I just don't want to get yelled at